src/detect-ipopts.c

```diff
@@ -32,10 +32,6 @@
 #include "detect-ipopts.h"
 #include "util-unittest.h"
 
-#define PARSE_REGEX "\\S[A-z]"
-
-static DetectParseRegex parse_regex;
```

Comment:
This PCRE removal could be its own commit ;-)

Reply:
I'm including it as part of the overhaul to make this detection function properly.